hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -5206,6 +5208,26 @@ private String reconfOzoneThreadNumberDirDeletion(String newVal) {
     return newVal;
   }
 
+  private String reconfFSSnapshotSSTFilteringServiceInterval(String newVal) {
```

Comment:
nit: please remove `FS` from the name (`reconfFS` reads hard, and it's not part of the config key anyway)